tangle-subxt/src/tangle_testnet_runtime.rs

```diff
@@ -57124,7 +57207,6 @@ pub mod api {
 					#[encode_as_type(
 						crate_path = ":: subxt :: ext :: subxt_core :: ext :: scale_encode"
 					)]
-					#[codec(dumb_trait_bound)]
```

Comment:
This line here @drewstone 
`#[codec(dumb_trait_bound)]` add this back and it should work.